### ISSUE TITLE
isis3varinit.py doesn't load environment variables for c-shell

### ISSUE THREAD

**Issue by jessemapel** [DOI-USGS/ISIS3#3134]
**ISIS version(s) affected**: 3.6.2

**Description**  

The isis3carinit.py script sets up the environment variables for bash, but not c-shell.

**How to reproduce**  

Run the script on an environment with isis3 installed. Then, activate the environment in c-shell. `ISISROOT`, `ISIS3DATA` and `ISIS3TESTDATA` are not set.

**Possible Solution**  

Can we just make a .csh version of the script that runs on activation?

**Additional context**

**Comment by jessemapel** [comment]
I was able to make c-shell work by making a `env_vars.csh` scripts in the `activate.d` and `deactivate.d` directories of the environment. Here's their contents.

## `activate.d/env_vars.csh`
```
#!/bin/csh
setenv ISISROOT "/usgs/cpkgs/anaconda3_linux/envs/isis3.6.2"
setenv ISIS3DATA "/usgs/cpkgs/isis3/data"
setenv ISIS3TESTDATA "/usgs/cpkgs/isis3/testData"
```
## `deactivate.d/env_vars.csh`
```
#!/bin/csh
unsetenv ISISROOT
unsetenv ISIS3DATA
unsetenv ISIS3TESTDATA
```

**Comment by jessemapel** [comment]
Now we need to make the `isis3varinit.py` script write both the bash and c-shell scripts.

**Comment by jessemapel** [comment]
Fixed by #3173
